### PR TITLE
feedback list

### PR DESCRIPTION
**Pull request by tinyels**

### CODE REVIEW

**Comment by tinyels** [comment]
The tests are not as complete as I would like and there is an issue with viewing the submitted feedback in a client dev build that I would like some help on.

File: client/src/shared/LoadingPlaceholder/index.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+
+const LoadingPlaceholder = () => <h2>Loading, please wait...</h2>;
```
Comment (macrael) [inline client/src/shared/LoadingPlaceholder/index.jsx]: Is the idea that the loading placeholder can be re-used most places we hit the server? neat.

File: client/src/scenes/SubmittedFeedback/IssueCards.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import LoadingPlaceholder from '../../shared/LoadingPlaceholder';
+import './IssueCards.css';
+
+const IssueCards = ({ issues }) => {
+  if (!issues) return <LoadingPlaceholder />;
+  if (issues.length === 0)
+    return <h2> There is no feedback at the moment! </h2>;
+  return (
+    <div className="issue-cards">
+      {(issues || []).map(issue => (
```
Comment (macrael) [inline client/src/scenes/SubmittedFeedback/IssueCards.jsx]: The || [] seems superfluous here, you are returning early for !issues above. It's not doing any harm, but maybe we don't need it.

File: client/src/shared/LoadingPlaceholder/index.jsx (hunk 1 above)
Comment (tinyels) [reply client/src/shared/LoadingPlaceholder/index.jsx]: Yeah, and made prettier. ;)

File: client/src/scenes/SubmittedFeedback/IssueCards.jsx (hunk 2 above)
Comment (tinyels) [reply client/src/scenes/SubmittedFeedback/IssueCards.jsx]: Yeah, that was there before the other checks. Cleaning up now!

**Pull request merged**